2023/11/20 11:46:04 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.20.0.2:5137/", host: "localhost"
2023/11/20 12:19:26 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:26 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:27 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:27 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:28 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:28 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:29 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:29 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:30 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:30 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files HTTP/1.1", upstream: "http://172.20.0.5:8080/files", host: "localhost", referrer: "https://localhost/"
2023/11/20 12:19:30 [error] 22#22: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /api/files/info/655b4734145337dc76c39b6a HTTP/1.1", upstream: "http://172.20.0.5:8080/files/info/655b4734145337dc76c39b6a", host: "localhost", referrer: "https://localhost/"
